docs/internals/deprecation.txt

```diff
@@ -32,6 +32,8 @@ details on these changes.
 
 * ``django.db.models.fields.add_lazy_relation`` will be removed.
 
+* ``skipIfCustomUser`` decorator located in ``django.contrib.auth.tests.utils`` will be removed.
```

Comment:
The django.contrib.auth.tests.utils.skipIfCustomUser decorator...
(and please wrap doc lines at 79 characters)